File: data/museum/oseg_generated/python/UpdateSpecialEvent_default_example.py
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    special_event_fields = models.SpecialEventFields()
    special_event_fields.name = "Pirate Coding Workshop"
    special_event_fields.location = "Computer Room"
    special_event_fields.eventDescription = "Captain Blackbeard shares his love of the C...language. And possibly Arrrrr (R lang)."
    special_event_fields.price = 25
    special_event_fields.dates = None

    try:
        api_caller = api.EventsApi(api_client)

        response = api_caller.update_special_event(
            event_id="dad4bce8-f5cb-4078-a211-995864315e39",
            special_event_fields=special_event_fields,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Events#update_special_event: %s\n" % e)

File: data/museum/oseg_generated/csharp/UpdateSpecialEvent_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class UpdateSpecialEventDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        var specialEventFields = new SpecialEventFields(
            name: "Pirate Coding Workshop",
            location: "Computer Room",
            eventDescription: "Captain Blackbeard shares his love of the C...language. And possibly Arrrrr (R lang).",
            price: 25,
            dates: null
        );

        try
        {
            var apiCaller = new EventsApi(config);

            var response = apiCaller.UpdateSpecialEvent(
                eventId: "dad4bce8-f5cb-4078-a211-995864315e39",
                specialEventFields: specialEventFields
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#UpdateSpecialEvent: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
